CONTRIBUTING.rst

```diff
@@ -1043,8 +1182,8 @@ as described in the static code checks documentation.
 Coding style and best practices
 ===============================
 
-Most of our coding style rules are enforced programmatically by ruff and mypy (which are run automatically
-on every pull request), but there are some rules that are not yet automated and are more Airflow specific or
+Most of our coding style rules are enforced programmatically by ruff and mypy , which are run automatically
+on every Pull Request (PR), but there are some rules that are not yet automated and are more Airflow specific or
 semantic than style
```

Comment:
Spacing and adding a reference to static checks.
```suggestion
Most of our coding style rules are enforced programmatically by ruff and mypy, which are run automatically
with static checks and on every Pull Request (PR), but there are some rules that are not yet automated and
are more Airflow specific or semantic than style.
```

Reply:
nice clarification.